ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroSRSchemaDataTranslator.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright 2021 Confluent Inc.
- *
- * Licensed under the Confluent Community License (the "License"); you may not use
- * this file except in compliance with the License.  You may obtain a copy of the
- * License at
- *
- * http://www.confluent.io/confluent-community-license
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations under the License.
- */
-
-package io.confluent.ksql.serde.avro;
-
-import io.confluent.ksql.serde.connect.ConnectSRSchemaDataTranslator;
-import io.confluent.ksql.util.KsqlException;
-import java.util.Optional;
-import org.apache.kafka.connect.data.Field;
-import org.apache.kafka.connect.data.Schema;
-import org.apache.kafka.connect.data.Struct;
-
-/**
- * Translates KSQL data and schemas to Avro equivalents.
- *
- * <p>Responsible for converting the KSQL schema to a version ready for connect to convert to an
- * avro schema.
- *
- * <p>This includes ensuring field names are valid Avro field names and that nested types do not
- * have name clashes.
- */
-public class AvroSRSchemaDataTranslator extends ConnectSRSchemaDataTranslator {
```

Comment:
Now I remember the reason I added just this for Avro might be `protobuf` doesn't support optional fields and `json_sr`'s field is always optional. So it's different I think. Can we avoid this refactor and just fix the schema name problem?